src/kernel/kernel.ts

```diff
@@ -2499,12 +2500,20 @@ export function Boot(fsType: string, fsArgs: any[], cb: BootCallback, args: Boot
 					cb(err, undefined);
 					return;
 				}
-				let writable = new bfs.FileSystem['InMemory']();
+				if (args.useLocalStorage) {
+					let writable = new bfs.FileSystem['LocalStorage']();
```

Comment:
let bindings are scoped to the block they are in -- they will not be visible after the if statement.  I think you need 

`let writable: bfs.FileSystem; // or whatever the type is`
before the `if`, and then just `writable = new bfs.FileSystem...` inside the `if`